.github/action-common-python-release/action.yml

```diff
@@ -42,17 +44,17 @@ runs:
     name: Maturin - Build
     uses: messense/maturin-action@v1
     with:
-      manylinux: '2_28'
+      manylinux: '2_34'
```

Comment:
This seemed to be the fix in https://github.com/rigetti/qcs-sdk-rust/actions/runs/13248276833